README.md

```diff
@@ -52,6 +52,15 @@ Once you have them you could try building them:
 cargo run --package fontc -- ../google_fonts_sources/sources/ofl/notosanskayahli/sources/NotoSansKayahLi.designspace
 ```
 
+### Building lots of fonts at once
```

Comment:
Can we be slightly more specific? What lots of fonts? I assume all Google Fonts for which we have sources?

Reply:
As per below I suggest we update this to be how to build all Google Fonts we have sources for and point to a script that walks through it, including the construction of the json file if a recent copy is not locally available.